ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using ASPNetCoreMastersTodoList.Api.BindingModels;
+using Microsoft.AspNetCore.Mvc;
+using Services.DTO;
+using Services.Interfaces;
+
+namespace ASPNetCoreMastersTodoList.Controllers
+{
+    public class ItemsController : ControllerBase
+    {
+        private readonly IItemService _itemService;
+        public ItemsController(IItemService itemService)
+        {
+            _itemService = itemService;
+        }
+        public int Get(int id)
+        {
+            return id;
+        }
+
+        public void Post(ItemCreateApiModel model)
```

Comment:
we need to use http verb Post in order to post data to the API. Please use [HttpPost]. And indicate the binding source such as [FromBody] as discussed during session 3

Reply:
updated